edx_solutions_api_integration/courses/views.py

```diff
@@ -1124,6 +1124,50 @@ def get_queryset(self):
         return users
 
 
+class CoursesUsersPassedList(SecureAPIView):
+    """
+    **Use Case**
+
+        CoursesUsersPassedList returns a collection of users passed in the course.
```

Comment:
I think we could have more appropriate return statement here something like this `CoursesUsersPassedList returns a list of user ids passed in the course`